README.md

```diff
@@ -4,22 +4,116 @@
 [![r2c Community Slack](https://img.shields.io/badge/r2c_slack-join-brightgreen?style=flat&logo=slack&labelColor=4A154B)](https://join.slack.com/t/r2c-community/shared_invite/enQtNjU0NDYzMjAwODY4LWE3NTg1MGNhYTAwMzk5ZGRhMjQ2MzVhNGJiZjI1ZWQ0NjQ2YWI4ZGY3OGViMGJjNzA4ODQ3MjEzOWExNjZlNTA)
 [![r2c Twitter](https://img.shields.io/twitter/follow/r2cdev?label=Follow%20r2cdev&style=social&color=blue)](https://twitter.com/intent/follow?screen_name=r2cdev)
 
-`semgrep` is a tool for easily detecting and preventing bugs and anti-patterns in
-your codebase. It combines the convenience of `grep` with the correctness of
-syntactical and semantic search. Developers, DevOps engineers, and security engineers
-use `semgrep` to write code with confidence.
+<p align="center">
+  <a href="#overview">Overview</a>
+  <span> · </span>
+  <a href="#installation">Installation</a>
+  <span> · </span>
+  <a href="#usage">Usage</a>
+  <br/>
+  <a href="#resources">Resources</a>
+  <span> · </span>
+  <a href="#contributing">Contributing</a>
+  <span> · </span>
+  <a href="#commercial-support">Commercial Support</a>
+</p>
+
+<h3 align="center">
+  A fast, lightweight static analysis tool that lets you search code like you’d write it.
+</h3>  
+<h3 align="center">
+  Find security bugs. Enforce coding standards. Scan every PR.
+</h3>
+<h3 align="center">  
+  Easily block bad code so you can get back to building the next great thing.
+</h3>
```

Comment:
IMO the styling and hierarchy of these three `<h3>`s make them seem like competing descriptions and/or taglines. I’d suggest either moving one to a top-level (above the badges, for example), or moving all three to a bullet list like:

With Semgrep, you can:

* search code like you’d write it
* find security bugs
* enforce coding standards
* scan every PR or build
* Easily block bad code so you can get back to building the next great thing.

Reply:
I had a similar reaction. Would you support me pulling the Bento README styling here, using the Semgrep logo, tagline, and your hierarchy changes?

![image](https://user-images.githubusercontent.com/605906/84199130-7b8ef880-aa59-11ea-915e-5a022e8cddc8.png)

What I like about how we laid out our other README is that navigation elements came before badges and other secondary information.

Concretely, I think I would use a tagline like:

> Fast and customizable static analysis for many languages. Find code matches the way you'd write them.

I'm trying to find a succinct way of emphasizing speed and simplicity. My suggestion needs work :)